src/components/ShareBar/ShareBar.tsx

```diff
@@ -18,11 +18,12 @@ export function ShareBar(p: Props) {
   const { share, total, className, warningThreshold, showPercentage } = p;
 
   const progress = useMemo(() => {
-    if (!share || !total || BigInt(total) == 0n) {
+    if (share == undefined || total == undefined || BigInt(total) == 0n) {
```

Comment:
shouldn't it be `===` instead of `==`?

Reply:
made this change intentionally during bigints update for safety. I'd keep it as it is just in case